internal/pkg/cli/storage_init_test.go

```diff
@@ -173,6 +174,30 @@ func TestStorageInitOpts_Validate(t *testing.T) {
 
 			wantedErr: errors.New("invalid engine type mysql: must be one of \"MySQL\", \"PostgreSQL\""),
 		},
+		"successfully validates aurora serverless version v1": {
+			mockWs:              func(m *mocks.MockwsAddonManager) {},
+			mockStore:           func(m *mocks.Mockstore) {},
+			inAppName:           "bowie",
+			inStorageType:       rdsStorageType,
+			inServerlessVersion: auroraServerlessVersionV1,
+			wantedErr:           nil,
+		},
+		"successfully validates aurora serverless version v2": {
+			mockWs:              func(m *mocks.MockwsAddonManager) {},
+			mockStore:           func(m *mocks.Mockstore) {},
+			inAppName:           "bowie",
+			inStorageType:       rdsStorageType,
+			inServerlessVersion: auroraServerlessVersionV2,
+			wantedErr:           nil,
```

Comment:
samesies!
```suggestion
```

Reply:
I removed `wantedErr: nil` in two test cases and left other test cases alone. Should I remove other `wantedErr: nil` or do so in another PR?